app/javascript/dashboard/routes/dashboard/Dashboard.vue

```diff
@@ -77,14 +79,41 @@ export default {
       const { show_secondary_sidebar: showSecondarySidebar } = this.uiSettings;
       return showSecondarySidebar;
     },
+    previouslyUsedDisplayType() {
+      const {
+        previously_used_conversation_display_type: conversationDisplayType,
+      } = this.uiSettings;
+      return conversationDisplayType;
+    },
+  },
+  watch: {
+    isDesktopView() {
+      const { LAYOUT_TYPES } = wootConstants;
+      this.updateUISettings({
+        conversation_display_type: !this.isDesktopView
+          ? LAYOUT_TYPES.EXPANDED
+          : this.previouslyUsedDisplayType,
+      });
+    },
   },
   mounted() {
+    this.handleResize();
+    window.addEventListener('resize', this.handleResize);
     bus.$on(BUS_EVENTS.TOGGLE_SIDEMENU, this.toggleSidebar);
```

Comment:
There should be a CSS-only way of doing this, I see we're hiding some elements, we can use `min-inline-size` to trigger these I guess (need to verify). Do we have transitions in the UI? Having an active listener can cause some performance issues for lower-end laptops and tablets. 

If a CSS-only way is not possible, we can perhaps throttle the event and see, it can potentially make the UI a bit janky though. If it does we can let the current changes stay as is, else we can consider throttling. 

Here's a possible way to throttle it

```js
let throttled = false,
const delay = 150 // 150ms
window.addEventListener('resize', function() {
  if (!throttled) {
    this.toggleSidebar();
    throttled = true;
    setTimeout(function() {
      throttled = false;
    }, delay);
  }
});
```

Reply:
@scmmishra I think we couldn't do it with CSS (correct me if I was wrong) because when the display breakpoint <= 992px, it will update the conversation display layout as 'expanded' to UISettings. We don't have transitions in UI. I will try it the other way that you mentioned. 
CC: @nithindavid

Reply:
Done https://github.com/chatwoot/chatwoot/pull/6134/commits/f84b05b16cdc66f1241555c71f00c4f555e9301e

Reply:
This won't remove the listener, since the function signature won't match with the one given originally, we can add the throttle in the handle resize itself

Another option is to write a throttle handler, it will return a throttled function

```js
function throttle(fn, delay) {
  let throttled = false;
  return function (...args) {
    if (throttled) {
      return;
    }
    throttled = true

    setTimeout(() => {
      throttled = false;
      fn(...args);
    }, delay);
  };
}
```

But for now, maybe we can just add the throttle to the `handleResize` itself. Upto you, on what seems better